### PR TITLE
Project with Tags

### LINKED ISSUE #54: Fetch project with tags
- Add query parameter to get project with list of tags.
API would look like the following: 
GET `/api/projects?withTags=True`
Example Output 
```
    {
        "id": 1,
        "name": "Google",
        "description": "Google LLC is an American multinational technology company that specializes in Internet-related services and products, which include online advertising technologies, search engine, cloud computing, software, and hardware.",
        "github": "https://github.com/google",
        "url": "https://google.com",
        "project_start_date": "1998-11-25T05:00:00.000Z",
        "image": "https://i.imgur.com/Hjpgzeq.jpg",
        "status": "approved",
        "createdAt": null,
        "updatedAt": null,
        "owner": {
            "name": "Sergey Brin",
            "username": "SergeyBrin"
        },
        tags: [ "Javascript", "Python", "Machine Learning"]
    }
```

### PR DESCRIPTION
**Pull request by F-Ting**
query param can be added to get tags associated with projects
fixes #54

### CODE REVIEW

File: node/server/controllers/projects.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ module.exports = {
 
   // list all projects
   list(req, res) {
-    return axios.get("http://localhost:8001/api/projects")
+    console.log(req.originalUrl)
```
Comment (bellkelly) [inline node/server/controllers/projects.js]: Remove logging?

**Review by bellkelly — COMMENTED** [review]
Does this work? For some reason, I'm getting errors when I test it, but I'm not sure if there's something weird on my end because I don't see an obvious issue in the code.

File: node/server/controllers/projects.js (hunk 1 above)
Comment (F-Ting) [reply node/server/controllers/projects.js]: good catch, I thought I removed that!

**Comment by F-Ting** [comment]
Well, the API is working on my end, however I need your confirmation on if the format returned make sense for you to display in the front-end.

**Comment by bellkelly** [comment]
Yup :) The format from the issue looks good to me! I'll try to figure out on my end why it's not working.

**Review by bellkelly — APPROVED** [review]

**Comment by F-Ting** [comment]
Actually if it is not working for you then I probably forgot something, lemme have another look.

**Pull request merged**